managed/resource_cluster.go

```diff
@@ -286,10 +292,26 @@ func createClusterSpec(ctx context.Context, apiClient *openapiclient.APIClient,
 	var diskSizeOK bool
 	var memoryMb int32
 	var memoryOK bool
+	var trackId string
+	var trackName string
+	var trackIdOK bool
 	var message string
 
 	networking := *openapiclient.NewNetworkingWithDefaults()
+
+	// Compute track ID for database version
 	softwareInfo := *openapiclient.NewSoftwareInfoWithDefaults()
+	if !plan.DatabaseVersion.Unknown {
+		trackName = plan.DatabaseVersion.Value
+	} else {
+		// Set track name to "Stable" by default
+		trackName = "Stable"
```

Comment:
Can you please try creating a free tier cluster? I think for free tier cluster we can only do Preview. I think if the user does not specify anything, we should not pass anything to the API.

Reply:
You are right, let me change this to not pass anything to the API is nothing is provided.